app/scripts/components/exploration/components/map/share-button.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useState } from 'react';
+import { Icon } from '@trussworks/react-uswds';
+
+import { SelectorButton } from '$components/common/map/style/button';
+import useThemedControl from '$components/common/map/controls/hooks/use-themed-control';
+import { useVedaUI } from '$context/veda-ui-provider';
+
+function ShareButtonComponent({
+  onClick,
+  disabled,
+  tipContent,
+  onMouseLeave
+}: {
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  disabled: boolean;
+  tipContent: string;
+  onMouseLeave?: () => void;
+}) {
+  return (
+    <SelectorButton
+      tipContent={tipContent}
+      tipProps={{ placement: 'left', showOnCreate: true, hideOnClick: false }}
+      disabled={disabled}
+      onClick={onClick}
+      onMouseLeave={onMouseLeave}
+    >
+      <Icon.Share size={3} />
+    </SelectorButton>
+  );
+}
+
+export function ShareButton(): JSX.Element | null {
+  const [linkCopied, setLinkCopied] = useState(false);
+  const { envUrlShortenerEndpoint } = useVedaUI();
+
+  const handleMouseLeave = useCallback(() => {
+    if (linkCopied) {
+      setLinkCopied(false);
+    }
+  }, [linkCopied]);
+
+  const handleShare = useCallback(async () => {
+    try {
+      // get current URL
+      const currentUrl = window.location.href;
+      const urlShortenerEndpoint = envUrlShortenerEndpoint;
+
+      if (urlShortenerEndpoint) {
+        try {
+          const response = await fetch(
+            `${urlShortenerEndpoint}?url=${encodeURIComponent(currentUrl)}`
+          );
```

Comment:
```suggestion

      if (envUrlShortenerEndpoint) {
        try {
          const response = await fetch(
            `${envUrlShortenerEndpoint}?url=${encodeURIComponent(currentUrl)}`
          );
```

I don't think this alias is necessary here.